verification/curator-service/Dockerfile

```diff
@@ -31,9 +32,27 @@ COPY verification/curator-service/ui/. .
 ENV REACT_APP_LOGIN_URL /auth/google
 RUN npm run-script build
 
-# Go back to curator service
-WORKDIR /usr/src/app/verification/curator-service/api
+
+# Multi-staged build, use a trimmed down version of node for prod.
+FROM node:alpine as prod
+
+# No need to run as root.
+USER node
```

Comment:
Any prerequisites to do this? On my machine, `groups node` is empty, so this will run as ~root anyway. Is the `node` user correctly configured in our to-be prod VMs?

Reply:
Not sure why it's not present on your machine, it is when I run the docker image:

```
docker run -it --rm --init node:12-alpine sh
/ # whoami
root
/ # su node
/ $ whoami
node
/ $ id -u node
1000
/ $ id -u root
0
```

Reply:
Yeah, this was my bad. WFM!